lib/cleanup/destroyable_records.rb

```diff
@@ -26,7 +26,7 @@ def print_data
 
     stdout.puts '********'
     stdout.puts 'Integration:'
-    stdout.puts integration.attributes.to_yaml
+    stdout.puts integration&.attributes.to_yaml
```

Comment:
In this instance, it might be nice to have a conditional

```ruby
if integration.nil?
 stdout.puts "No associated integration"
else 
  stdout.puts integration.attributes.to_yaml
end
```

Reply:
Added!